turbinia/lib/utils.py

```diff
@@ -97,6 +97,43 @@ def extract_artifacts(artifact_names, disk_path, output_dir, credentials=[]):
   return _image_export(image_export_cmd, output_dir)
 
 
+def extract_custom_artifacts(artifact_names, artifact_definition_file,
+                             disk_path, output_dir, credentials=[]):
```

Comment:
Looks like `yapf` wants this with a newline after the method name and open parenthesis (that's why one of the tests is failing).